Makefile

```diff
@@ -0,0 +1,14 @@
+SHELL=bash
+
+.PHONY: rpc
+
+PROTO_IN = $(shell find proto -name '*.proto')
+
```

Comment:
pls no Makefile, we've evolved to `just`

Reply:
haha, oki, what the hell is just

Reply:
https://github.com/casey/just

Reply:
Simple `make` alternative created by the ordinals guy